### PR TITLE
Release 3.0.17

### PR DESCRIPTION
**Pull request by erick0z**
- Update repository to match Clear Containers release: 3.0.17

This PR also have extra commits to fix some minor issues:

 - Fix runtime's versions.txt URL
 - proxy: remove postscript warnings for suse and opensuse
 - image: remove unnecesary preinstall task

### CODE REVIEW

File: clear-containers-image/update_image.sh [hunk 1]
```diff
@@ -11,8 +11,14 @@ source ../versions.txt
 source ../scripts/pkglib.sh
 
 # get versions from clearcontainers/runtime repository
-runtime_versions_url="https://raw.githubusercontent.com/clearcontainers/runtime/master/versions.txt"
-source <(curl -sL "${runtime_versions_url}")
+latest_runtime_tag=$(curl -s https://github.com/clearcontainers/runtime/releases/latest | cut -d / -f8 | cut -d '"' -f1)
+if [ -n "$latest_runtime_tag" ] && [[ "$latest_runtime_tag" =~ ^[0-9]+\.[0-9]{0,2}+\.[0-9]{0,3}$  ]]; then
```
Comment (jodh-intel) [inline clear-containers-image/update_image.sh]: This pattern won't match all valid semver versions (https://semver.org/). I think the most pragmatic fix here is simply to drop the trailing `$` anchor so this just matches on a valid prefix:

```
$ latest_runtime_tag=3.0.17
$ [[ "$latest_runtime_tag" =~ ^[0-9]+\.[0-9]{0,2}+\.[0-9]{0,3}  ]] && echo ok
ok
$ latest_runtime_tag=99.99.999-1.2.3.alpha.foo+git03b38063b27b960e26d3cfab92ede6aaf26f65b3.wibble.9.8.7foo
[[ "$latest_runtime_tag" =~ ^[0-9]+\.[0-9]{0,2}+\.[0-9]{0,3}  ]] && echo ok
ok
```

**Comment by erick0z** [comment]
@jodh-intel done, thanks!

**Comment by jodh-intel** [comment]
Thanks @erick0z.

lgtm

**Comment by jcvenegas** [comment]
lgtm

**Review by jcvenegas — APPROVED** [review]

**Pull request merged**